docs/multi-creds-assignment.rst

```diff
@@ -0,0 +1,110 @@
+.. _ag_multicred_assgn:
+
+Multi-Credential Assignment
+=============================
+
+.. index::
+   single: credentials
+   pair: credentials; multi
+   pair: credentials; assignment
```

Comment:
What does this do?

Reply:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#index-generating-markup implies this is for some references but I don't see where it's used...

Reply:
Our controller docs used to live in the product-docs repo and there were two outputs: html and PDF. These index entries generated the index for the PDF output.  
![image](https://github.com/user-attachments/assets/bc149062-e818-4f73-97f6-6536399e6f4c)
These were migrated directly from awx, which was directly migrated from the product-docs repo. Since there is no PDF output offered for these upstream projects, I can remove these as they are no longer used.